.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: xenial
```

Comment:
we might want to look into testing this on a windows dist if that's an option

Reply:
Yeah I added multiple `jobs` to the `.travis.yml` file to include two `os` args: `xenial` and `windows`. We can probably remove `xenial` since I don't think anyone will be running these notebooks on non-Windows machines.

Reply:
UPDATE: nvm, `python` is unsupported on the Windows dist, reverting back to `dist: xenial`.